tests/test_needs_warning.py

```diff
@@ -0,0 +1,14 @@
+from sphinx_testing import with_app
+
+
+@with_app(buildername="html", srcdir="doc_test/doc_needs_warnings")  # , warningiserror=True)
+def test_needs_warnings(app, status, warning):
+    app.build()
+
+    # stdout warnings
+    warnings = warning.getvalue()
+
+    # check warnings contents
+    assert "WARNING:   invalid_status: failed" in warnings
```

Comment:
2 spaces, is this a bug in the code? Maybe fix?

Reply:
yes, 2 spaces are in the code, like: 

logger.info("  {}: failed".format(warning_name))

will remove the 1 space.